src/components/Layout/index.jsx

```diff
@@ -25,16 +25,16 @@ const Layout = ({ children }) => {
   const links = [
     {
       component: (
-        <S.Link to="">
+        <S.Link to="home">
           <HomeIcon />
         </S.Link>
       ),
-      isLogin: 'general',
+      isLogin: 'loggedIn',
       key: 'home'
     },
     {
       component: (
-        <S.Link to="registration">
+        <S.Link to="">
```

Comment:
it was better when Registration page had /registration path and Home page had "/" path